process_manager/templates/process_manager/partials/process_table.html

```diff
@@ -1,4 +1,2 @@
 {% load render_table from django_tables2 %}
-<div hx-get="{% url 'process_manager:process_table' %}"
-     hx-trigger="load delay:1s"
-     hx-swap="outerHTML">{% render_table table %}</div>
+<div id="table-container">{% render_table table %}</div>
```

Comment:
```suggestion
{% render_table table %}
```
With `innerHTML` -- see other comment -- we only need `render_table` here.